typescript/index.d.ts

```diff
@@ -180,7 +180,8 @@ declare namespace WoT {
         data?: any;
     }
 
-    export type PropertyMap = object | { [key: string]: any; };
+    export type PropertyReadMap = Map<string,InteractionOutput>
+    export type PropertyWriteMap = Map<string,InteractionInput>
```

Comment:
nits:
* spaces between arguments -> `Map<string, InteractionOutput>`
* `;` in the end to be consistent with the rest?

Reply:
Should be solved now